src/platform/text-editor/text-editor.component.ts

```diff
@@ -110,4 +110,84 @@ export class TdTextEditorComponent implements AfterViewInit, ControlValueAccesso
   toTextArea(): void {
     this._simpleMDE.toTextArea();
   }
+
+  toggleBold(): void {
```

Comment:
Wouldnt be easier to expose the `_simpleMDE` object, rather than proxying every single method?

Reply:
Pushed in change to expose simpleMDE object in a getter